src/Farmer/Arm/Network.fs

```diff
@@ -113,8 +113,8 @@ type VirtualNetworkGateway =
                                    properties =
                                     let allocationMethod, ip =
                                         match ipConfig.PrivateIpAllocationMethod with
-                                        | DynamicPrivateIp -> "Dynamic", null
-                                        | StaticPrivateIp ip -> "Static", string ip
+                                        | PrivateIpAddress.DynamicPrivateIp -> "Dynamic", null
```

Comment:
Since these are fully qualified now, perhaps the two values / cases should just be `Dynamic` and `Static` i.e. `PrivateIpAddress.Dynamic` etc.